src/modules/HomePage/HomePage.tsx

```diff
@@ -1,9 +1,53 @@
+/* eslint-disable max-len */
+import React from 'react';
+import { PhoneCard, PhoneItem } from '../shared/components/PhoneCard';
 import './HomePage.module.scss';
 
-import React from 'react';
+const phone: PhoneItem = {
```

Comment:
Use this only for checking your card locally, no need to push it

Reply:
Yes of course, it was only a temporary push